modules/custom/az_publication/az_publication.module

```diff
@@ -861,6 +861,36 @@ function template_preprocess_az_inline_entity_role_form_entity_table(array &$var
   }
 }
 
+/**
+ * Implements hook_xmlsitemap_link_alter().
+ */
+function az_publication_xmlsitemap_link_alter(array &$link, array $context) {
+
+  // Modify publication links to not appear if they are noncanonical.
+  // This behavior is skipped for overridden links.
+  if (!empty($context['entity']) && $context['entity'] instanceof NodeInterface && $link['access'] && ($context['entity']->bundle() === 'az_publication') && (empty($link['status_override']))) {
+    // Fields that represent a canonical copy of the publication exists.
+    // @todo determine list.
+    $canonicals = [
+      'field_az_publication_doi',
+      'field_az_publication_isbn',
+      'field_az_publication_issn',
+      'field_az_publication_link',
+      'field_az_publication_pmcid',
+      'field_az_publication_pmid',
```

Comment:
Additions or subtractions to this list welcomed - first draft attempt.

Reply:
> Additions or subtractions to this list welcomed - first draft attempt.

I thought your suggestion from the discussion last week that this could probably just be `field_az_publication_doi` and `field_az_publication_link` seemed reasonable.